migrations/sql/V2020.04.01.10.33__add_now_statuses.sql

```diff
@@ -0,0 +1,23 @@
+-- Create a new column for defining the desired display order of the statuses
+ALTER TABLE now_application_status ADD COLUMN display_order integer;
+
+-- Inactivate all of the existing Core statuses and set their display order
+UPDATE now_application_status SET active_ind = false, display_order = 5 WHERE now_application_status_code = 'ACC';
```

Comment:
I agree. If they're not in production, I guess we don't need an ETL.
Also, I already adjusted our factories to only generate test data with the active statuses.
Looks like the default status is one of the old ones, `UNR`:
![image](https://user-images.githubusercontent.com/17113094/78302993-b96b2000-74f0-11ea-93ba-87b8e9137dee.png)
Maybe it should be null (if it's a nullable field), otherwise... What's a new good default given the new statuses?